internal/dependency/client_set.go

```diff
@@ -277,6 +277,7 @@ func newTransport(i *CreateClientInput) (*http.Transport, error) {
 			KeepAlive: i.TransportDialKeepAlive,
 		}).Dial,
 		DisableKeepAlives:   i.TransportDisableKeepAlives,
+		ForceAttemptHTTP2:   true,
```

Comment:
[http docs](https://golang.org/pkg/net/http/#Transport)
> // To use a custom dialer or TLS config and still attempt HTTP/2
    // upgrades, set this to true.
    ForceAttemptHTTP2 bool // Go 1.13

This line does so much, great find in the docs to confirm that hcat wasn't using http2 but could.

Reply:
Yes and it is not as clear about it as it could be. The http.Transport docs say ... 

> To explicitly enable HTTP/2 on a transport, use golang.org/x/net/http2 and call ConfigureTransport.

That ConfigureTransport method does the exact same thing as setting ForceAttemptHTTP2, but after the fact. So if you have an already configured Transport and want it to support HTTP2 you call that. If you know at configuration time that you want it you just set the ForceAttemptHTTP2 variable. Had to compare the source of the 2 code bases to be sure I was doing the right thing. SMH.